File: Modul-2/C/PRAK203_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    float a, b, i, j, x, y, hasil;
 
    printf("Masukkan nilai a, b, i, j, x, y secara berurutan: ");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    hasil = (((a - b) *  i) / j) - (x + y);

    printf("Hasil dari penghitungan adalah: %.3f", hasil);

    return 0;
}

File: Modul-2/PYTHON/PRAK203_2410817210001_MuhammadFadillah.py
a, b, i, j, x, y = map(float, input("Masukkan nilai a, b, i, j, x, y secara berurutan: ").split())

hasil = (((a - b) *  i) / j) - (x + y)

print(f"Hasil dari perhitungan adalah: {hasil:.3f}")